.github/workflows/lint.yml

```diff
@@ -17,22 +17,13 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           go-version: 1.19
-          check-latest: true
       - uses: actions/checkout@v3
-      - uses: technote-space/get-diff-action@v6.1.1
-        with:
-          PATTERNS: |
-            **/**.go
-            go.mod
-            go.sum
       - uses: golangci/golangci-lint-action@v3.3.0
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
           version: latest
           args: --timeout 10m
           github-token: ${{ secrets.github_token }}
-        # Check only if there are differences in the source code
-        if: "env.GIT_DIFF"
```

Comment:
What's the reasoning behind removing the check for the differences in the source code? As well as removing the get-diff-action? Think that it's used in most of the other linters.